### ISSUE TITLE
Terminal issue 

### ISSUE THREAD

**Issue by exhelios** [Magisk-Modules-Repo/Systemless-keweon-DNS-Manager#2]
When I type keweon in terminal as root it say nothing found for keweon, I tried it with 3 different terminal. Any solution ?
Thanks

**Comment by bhaskarjy0ti** [comment]
Have you updated it to v1.4 ?
Check if /sbin/.core/img/keweon-DNS exist !

**Comment by exhelios** [comment]
@bhaskarjy0ti yes 1.4 is here and all files in the directory (/sbin/.corr/img/keweon)
Also here is the output for the command in two different terminal :
/data/data/com.termux/files/usr/bin/login
/system/bin/sh

**Comment by bhaskarjy0ti** [comment]
Perform a clean install of the module. From v1.3 onwards keweon is in  /sbin/.core/img/keweon-DNS .
And use the terminal with output "/system/bin/sh"
